orga/tests/integration/components/participant/assessment/tabs_test.js

```diff
@@ -0,0 +1,25 @@
+import { module, test } from 'qunit';
+import setupIntlRenderingTest from '../../../../helpers/setup-intl-rendering';
+import { render } from '@ember/test-helpers';
+import hbs from 'htmlbars-inline-precompile';
+
+module('Integration | Component | Participant::Assessment::Tabs', function(hooks) {
+  setupIntlRenderingTest(hooks);
+
+  hooks.beforeEach(function() {
+    this.owner.setupRouter();
+  });
+
+  test('it should navigate', async function(assert) {
```

Comment:
**suggestion (non-blocking) :** `it displays navigation to Resultat and to Analyse`

Reply:
done